src/stateMachines/maintenance/syncDeviceInfo.ts

```diff
@@ -0,0 +1,108 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2022
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+import { assign, createMachine } from 'xstate'
+import {
+  coalesceMessage,
+  commonContext,
+  type CommonMaintenanceContext,
+  HttpResponseError
+} from './common'
+import Logger from '../../Logger'
+import * as Task from './doneResponse'
+import { devices } from '../../WebSocketListener'
+import got from 'got'
+import { Environment } from '../../utils/Environment'
+
+export interface DeviceInfo {
+  ver: string
+  build: string
+  sku: string
+  currentMode: string
+  features: string
+  ipConfiguration: { ipAddress: string }
+}
+
+export const SyncDeviceInfoEventType = 'SYNC_DEVICE_INFO'
+export type SyncDeviceInfoEvent =
```

Comment:
this event definition rather than have payload of any, should be deviceInfo: DeviceInfo

Reply:
fixed